python-package/lightgbm/dask.py

```diff
@@ -232,6 +298,12 @@ def _train(
         sum(group) = n_samples.
         For example, if you have a 100-document dataset with ``group = [10, 20, 40, 10, 10, 10]``, that means that you have 6 groups,
         where the first 10 records are in the first group, records 11-30 are in the second group, records 31-70 are in the third group, etc.
+    eval_set : List of (X, y) tuples of Dask data collections, or None, optional (default=None)
+        List of (X, y) tuple pairs to use as validation sets.
+    eval_sample_weight: List of Dask data collections or None, optional (default=None)
+        List of dask Array or dask Series, weights for each validation set in eval_set.
```

Comment:
```suggestion
        List of Dask Array or Dask Series, weights for each validation set in eval_set.
```

We try to use "Dask" (capitalized) when referring to the package and "dask" (lowercase) when referring to a module in the library, based on https://marketing.dask.org/en/latest/refer.html

Reply:
TIL!

Reply:
my teammate @rikturr taught me about this last week 😀